py-polars/polars/dataframe/frame.py

```diff
@@ -6185,6 +6185,15 @@ def drop_nans(
             (default), use all columns (note that only floating-point columns
             can contain NaNs).
 
+        Warnings
```

Comment:
Can we changes these to Notes?

Reply:
Spooky... I was about to request the exact same change 🤣 (As warning that a function will behave exactly as advertised seems a bit alarmist, hah).